e2e_test/iceberg/test_case/iceberg_source_all_delete.slt

```diff
@@ -0,0 +1,96 @@
+statement ok
+set streaming_parallelism=4;
+
+statement ok
+CREATE TABLE s1 (i1 int, i2 varchar, i3 varchar);
+
+statement ok
+CREATE MATERIALIZED VIEW mv1 AS SELECT * FROM s1;
+
+statement ok
+CREATE SINK sink1 AS select * from mv1 WITH (
```

Comment:
Please add some comments to illustrate how position delet and equality files are generated and the sink is assumed to be decoupled.